src/platform/datapath_raw_xdp_win.c

```diff
@@ -1346,11 +1351,15 @@ CxPlatSocketUpdateQeo(
 
     BOOLEAN AtLeastOneSucceeded = FALSE;
     for (CXPLAT_LIST_ENTRY* Entry = Xdp->Interfaces.Flink; Entry != &Xdp->Interfaces; Entry = Entry->Flink) {
-        Status =
-            Xdp->XdpApi->XdpQeoSet(
-                CONTAINING_RECORD(Entry, XDP_INTERFACE, Link)->XdpHandle,
-                Connections,
-                sizeof(Connections));
+        if (Xdp->XdpQeoSet != NULL) {
+            Status =
+                Xdp->XdpQeoSet(
```

Comment:
Does `XdpQeoSet` ever return `E_NOINTERFACE` itself? I'm just wondering if we can differentiate between not available or the call failed.